### ISSUE TITLE
infoseek_test_withkb.json for test set

### ISSUE THREAD

**Issue by duyc168** [edchengg/infoseek_eval#3]
Hi, I find that for training set and validation set, there are infoseek_$split_withkb.json files that link the images to the text content in wikidata6M, but I do not find infoseek_test_withkb.json, neither on github or on https://huggingface.co/datasets/ychenNLP/oven.  Could you provide the infoseek_test_withkb.json file? thanks ~

**Comment by edchengg** [comment]
Hi @duyc168,
Thanks a lot for your interests!
We released withkb information for train/valid for model development.
For test set, the goal is to retrieve entity directly from the wikipedia. So we don't plan to release the gold withkb information publicly.
